ui/src/message_popup/experience_page.jsx

```diff
@@ -110,12 +118,25 @@ export default React.createClass({
       helpType,
       sessionLength
     } = this.state;
+    if (_.has(this.props.query, 'mobilePrototype')) return this.renderMobilePrototype();
     if (!hasStarted) return this.renderInstructions();
     if (questionsAnswered >= sessionLength) return this.renderDone();
 
     const question = questions[questionsAnswered];
     return (
       <div style={styles.container}>
+        <AppBar 
```

Comment:
The `AppBar` looks like it's here twice, you can factor it out so that the AppBar is in `render`, and then whatever is inside it changes based on what state and screen we're in (eg., instructions screen, question screen).